chore: update README content (#1618)
Qovery/console: README.md

## Contributing

> Qovery Console is in its early stage of development and we need some help, you are welcome to contribute! You can propose improvements directly from the [issues](https://github.com/Qovery/console/issues) page or include them in your pull request for changes.

Reviewer: ```suggestion
Qovery Console is actively develop and we need some help, you are welcome to contribute! You can propose improvements directly from the [issues](https://github.com/Qovery/console/issues) page or include them in your pull request for changes.
```

> ### Why do we use NX?

Reviewer: ```suggestion
### Why do we use Nx?
```
